nix/postgres/default.nix

```diff
@@ -56,6 +56,14 @@ in
       description = "The DB socket directory";
     };
 
+    # Based on: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS
+    connectionURI = lib.mkOption {
+      type = lib.types.functionTo lib.types.str;
+      readOnly = true;
+      default = dbName: "postgres://${config.listen_addresses}:${builtins.toString config.port}/${dbName}";
```

Comment:
I can imagine being able to pass additional arguments in the future. So the arguments should probably be an attrset.

See https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS

Reply:
Here you go: https://github.com/juspay/services-flake/pull/146